album-recommendation-java/src/main/application/services.xml

```diff
@@ -2,9 +2,18 @@
 <!-- Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
 <services version="1.0" xmlns:deploy="vespa" xmlns:preprocess="properties">
 
+  <admin version="2.0">
+    <adminserver hostalias="node1" />
+    <metrics>
+      <consumer id="my-metric-set">
```

Comment:
I suggest to call it 'my-metrics' to avoid even more confusion between consumer and metric-set.

Reply:
agreed